File: Algo/Common/MinMax/minMax.py
def minMax(arr):
    min = arr[0]
    max = arr[0]
    for i in arr:
        if( i > max):
            max = i
        if( i < min):
            min = i
    return min, max

arr = [1 , 3 , 7 , 8 , 12 , 13 , 17 , 20 , 24]
print(minMax(arr))

File: Algo/Common/MinMax/MinMax.java
class MinMax{  

    public static void main(String args[]){ 
        int[] arr = new int[] {3,7,2,8,3};
        minMax(arr);
    }  

    public static void minMax(int[] arr) {
        int min = arr[0];
        int max = arr[0];
        for(int value: arr) {
            if (value > max) {
                max = value;
            }
            if (value < min) {
                min = value;
            }
        }

        System.out.println("Min is " + min);
        System.out.println("Max is " + max);
    } 
} 